src/codebert/codebert_model.rs

```diff
@@ -0,0 +1,360 @@
+    );
+    /// Shared under Apache 2.0 license by the Hugging Face Inc. team at <https://huggingface.co/huggingface/CodeBERTa-language-id>. Modified with conversion to C-array format.
+    pub const CODEBERTA_LANG: (&'static str, &'static str) = (
+        "codeberta-language-id/config",
+        "https://huggingface.co/huggingface/CodeBERTa-language-id/resolve/main/config.json",
+    );
+    /// Shared under Apache 2.0 license by the Microsoft team at <https://github.com/microsoft/CodeBERT>. Modified with conversion to C-array format.
+    pub const CODEBERT_MLM: (&'static str, &'static str) = (
+        "codebert-mlm/model",
+        "https://huggingface.co/microsoft/codebert-base-mlm/resolve/main/config.json",
+    );
+}
+
+impl CodeBertVocabResources {
+    /// Shared under Apache 2.0 license by the Microsoft team at <https://github.com/microsoft/CodeBERT>. Modified with conversion to C-array format.
+    pub const CODEBERT: (&'static str, &'static str) = (
+        "codebert/vocab",
+        "https://huggingface.co/microsoft/codebert-base/resolve/main/vocab.json",
+    );
+    /// Shared under Apache 2.0 license by the Hugging Face Inc. team at <https://huggingface.co/huggingface/CodeBERTa-language-id>. Modified with conversion to C-array format.
+    pub const CODEBERTA_LANG: (&'static str, &'static str) = (
+        "codeberta-language-id/vocab",
+        "https://huggingface.co/huggingface/CodeBERTa-language-id/resolve/main/vocab.json",
+    );
+    /// Shared under Apache 2.0 license by the Microsoft team at <https://github.com/microsoft/CodeBERT>. Modified with conversion to C-array format.
+    pub const CODEBERT_MLM: (&'static str, &'static str) = (
+        "codebert-mlm/model",
+        "https://huggingface.co/microsoft/codebert-base-mlm/resolve/main/vocab.json",
+    );
+}
+
+impl CodeBertMergesResources {
+    /// Shared under Apache 2.0 license by the Microsoft team at <https://github.com/microsoft/CodeBERT>. Modified with conversion to C-array format.
+    pub const CODEBERT: (&'static str, &'static str) = (
+        "codebert/merges",
+        "https://huggingface.co/microsoft/codebert-base/resolve/main/merges.txt",
+    );
+    /// Shared under Apache 2.0 license by the Hugging Face Inc. team at <https://huggingface.co/huggingface/CodeBERTa-language-id>. Modified with conversion to C-array format.
+    pub const CODEBERTA_LANG: (&'static str, &'static str) = (
+        "codeberta-language-id/merges",
+        "https://huggingface.co/huggingface/CodeBERTa-language-id/resolve/main/merges.txt",
+    );
+    /// Shared under Apache 2.0 license by the Microsoft team at <https://github.com/microsoft/CodeBERT>. Modified with conversion to C-array format.
+    pub const CODEBERT_MLM: (&'static str, &'static str) = (
+        "codebert-mlm/model",
+        "https://huggingface.co/microsoft/codebert-base-mlm/resolve/main/merges.txt",
+    );
+}
+
+pub struct CodeBertForFeatureExtraction {
```

Comment:
I believe you could declare an alias as in https://github.com/guillaume-be/rust-bert/blob/59c0e668d68eb80c674100049ecd37c35c9ab4b5/src/roberta/roberta_model.rs#L997